asmcomp/power/emit.mlp

```diff
@@ -133,8 +134,12 @@ let emit_reg r =
 
 let emit_stack env r =
   match r.loc with
+  | Stack (Domainstate n) ->
+      let ofs = n + Domainstate.(idx_of_field Domain_extra_params) * 8 in
+      `{emit_int ofs}(30)`
```

Comment:
The comment at the start of `proc.ml` associates the domain state pointer to register `28`.

Reply:
That's not what I see:
https://github.com/ocaml/ocaml/blob/4e9eb755e60d0f0f4e458eb3f7af26bce5d8e2ba/asmcomp/power/proc.ml#L39